### DISCUSSION TITLE
Payload admin dashboard sorting does not work after version 1.3.0 to 1.6.11 upgrade

### DISCUSSION THREAD

**Discussion by dberhane** [payloadcms/payload#2121] [General]
I have recently upgraded Payloadcms from version 1.3.0 to 1.6.11 and read and followed the changelog carefully. 
I have fixed most of my issues. The only thing I could not fix is that the Payload admin dashboard functionality does not seem to be working for me. When I try to sort a collection by title or date, it does not work. If I downgrade to v 1.3.0, it works fine. Am I missing any config tweaks in the migration process? I would be grateful for your help.

**Comment by JessRynkar** [comment]
Hi @dberhane - what is the behavior you're seeing when you try to sort? does it break the page or do nothing? and if you get an error in your console or terminal, could you please share it.

**Reply by dberhane** [reply]
Thank @JessChowdhury for your response. 

When I click on the up and down arrows, it does not break the page, and there is no error message on the browser Console as well. 

The only thing I see is XHR POST & GET requests.

For example, when I click on the "Title" up or down arrow, I see the following XHR calls:

POST: https://xx.mydomain/api/_preferences/articlemetadata-list
GET: https://xx.mydomain/api/articlemetadata?locale=en&depth=0&draft=true&sort=title

**Reply by dberhane** [reply]
Thank you very much @JessChowdhury for asking the right question. I was able to troubleshoot the problem by looking at my response to you above and doing some debugging. The XHR GET request above has query param (draft=true) that was the cause of the issue.  I had to review and fix the collection definition that allowed me to resolve the admin sorting issue.

**Reply by JessRynkar** [reply]
💥  Nice catch @dberhane!

**Reply by lilnotes** [reply]
hey, guys
I got the same issue and 
```
if(args.args.draft===true) {
          args.args.draft=undefined
}
```
in beforeOperationHook fixed it for me, but are you going to fix it inside PayloadCMS?

**Reply by jmikrut** [reply]
Hey @lilnotes — I just pushed a fix for this. Expect a release shortly and then you will be all set.

Thank you!
